### PR TITLE
Fix: Issues Addressed (Aug 5 - Aug 9)

### PR DESCRIPTION
**Pull request by MiteshDarda**
## Description

Issues Addressed (Aug 5 - Aug 9):

Fixes:

	•	Aligned the sparrow icon correctly on the default screen.
	•	Updated the drop-shadow effect for AI suggestions icons.
	•	Adjusted the position of the “(… Generating)” text in the AI chat box.
	•	Improved the hover state of the “Send Message” icon.
	•	Fixed the UI of the document input box.

Features:

	•	In AI chat, replaced the copy icon with a tick icon for 5 seconds after copying.
	•	Added new animations for the appearance and disappearance of the AI chat.

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

File: src/packages/@app/styles/style.scss [hunk 1]
```diff
@@ -482,6 +484,8 @@ $theme-red-2B1C1D: #2B1C1D;
   --icon-danger-800 : #{$theme-red-f33346};
   --icon-danger-900 : #{$theme-red-fe2e2e};
   --icon-danger-1000 : #{$theme-red-312325};
+
+  --icon-disabled-100 : #{$theme-grey-333333};
 }
```
Comment (itsmdasifraza) [inline src/packages/@app/styles/style.scss:489]: this is not following the existing naming pattern

File: src/packages/@library/ui/tooltip/Tooltip.svelte [hunk 2]
```diff
@@ -20,6 +26,11 @@
   let isDelayed = false;
 
   const toggleTooltip = () => {
+    if (title === "") {
+      show = false;
+    } else {
+      show = true;
+    }
```
Comment (itsmdasifraza) [inline src/packages/@library/ui/tooltip/Tooltip.svelte:33]: cant change `show` variable in library module, this should be handled from the app components only as a props, this variable holds some kind of accessibility for guest user.

File: src/packages/@workspaces/features/chat-bot/assests/Tick.png
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/chat-bot/assests/Tick.png:1]: tick.png not required lets migrate it to svg

File: src/packages/@workspaces/features/chat-bot/components/chat-item/ChatItem.svelte [hunk 3]
```diff
@@ -3,6 +3,8 @@
   import { marked } from "marked";
   import { notifications } from "@library/ui/toast/Toast";
   import { copyIcon } from "../../assests";
+  import tickIcon from "../../assests/tick.png";
+  import { tick } from "svelte";
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/chat-bot/components/chat-item/ChatItem.svelte:7]: import tickIcon from index as used or copy Icon

File: src/packages/@workspaces/features/chat-bot/components/prompt-input/PromptInput.svelte [hunk 4]
```diff
@@ -8,6 +8,16 @@
   export let prompt: string = "";
   export let onUpdateAiPrompt;
   export let isResponseGenerating;
+
+  let isHovered = false;
+
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/chat-bot/components/prompt-input/PromptInput.svelte:13]: let make it more descriptive, for example `isSendButtonHovered`

File: src/packages/@workspaces/features/chat-bot/components/prompt-input/PromptInput.svelte [hunk 5]
```diff
@@ -8,6 +8,16 @@
...
+  function handleMouseEnter() {
+    isHovered = true;
+  }
+
+  function handleMouseLeave() {
+    isHovered = false;
+  }
 </script>
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/chat-bot/components/prompt-input/PromptInput.svelte:22]: use arrow function, we are following es6 patterns

File: src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte [hunk 6]
```diff
@@ -179,13 +181,22 @@
       onUpdateRequestState({ requestSplitterDirection: e.detail });
     }}
   /> -->
-  <Tooltip title={"Save"} placement={"bottom"} distance={12} zIndex={10}>
+  <Tooltip
+    title={isGuestUser ? "Login to save" : "Save"}
+    show={requestUrl !== ""}
+    placement={"bottom-left"}
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte:187]: guest user can also save, and I think requestUrl has not any dependency to show or hide tooltip
cc: @nagesh953 @Monika-APM @astitva-techdome

File: src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte [hunk 7]
```diff
@@ -179,13 +181,22 @@
...
       on:mouseenter={handleMouseEnter}
       on:mouseleave={handleMouseLeave}
-      disabled={isSave || userRole === WorkspaceRole.WORKSPACE_VIEWER
+      disabled={isSave ||
+      requestUrl === "" ||
+      userRole === WorkspaceRole.WORKSPACE_VIEWER ||
+      isGuestUser
         ? true
         : false}
          style="background-color: {isSave ? 'var(--icon-secondary-550)' : 'var(--bg-secondary-400)'}; color: white;"
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte:202]: api save button cant be dependent on request url or guest user 
cc: @nagesh953 @Monika-APM  @astitva-techdome

File: src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte [hunk 8]
```diff
@@ -197,6 +208,8 @@
           ? "var(--icon-secondary-380)"
           : isHovered && !isSave && !isGuestUser
           ? "var(--icon-primary-200)"
+          : isGuestUser || requestUrl === ""
+          ? "var(--icon-disabled-100)"
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/rest-explorer/components/http-url-section/HttpUrlSection.svelte:212]: color doesnt have any dependency on guest user or request url
cc: @nagesh953 @Monika-APM  @astitva-techdome

File: src/packages/@workspaces/features/rest-explorer/components/response-default-screen/ResponseDefaultScreen.svelte [hunk 9]
```diff
@@ -112,4 +112,15 @@
...
+
+  .overflow-show-on-hover {
+    overflow-y: hidden;
+    transition: overflow 0.3s ease-in-out;
+    padding-right: 10px;
+  }
+
+  .overflow-show-on-hover:hover {
+    padding-right: 0;
+    overflow-y: auto;
```
Comment (itsmdasifraza) [inline src/packages/@workspaces/features/rest-explorer/components/response-default-screen/ResponseDefaultScreen.svelte:124]: lets work on deciding class names, haha

**Review by itsmdasifraza — CHANGES_REQUESTED** [review]

**Review by Astitva877 — CHANGES_REQUESTED** [review]
Please resolve the merge conflict

**Review by itsmdasifraza — APPROVED** [review]

**Review by itsmdasifraza — APPROVED** [review]

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Review by Astitva877 — APPROVED** [review]

**Pull request merged**